Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM node:8-alpine
+FROM node:8-alpine as build
 
-# Setup directory
 WORKDIR /app
 
-# Add/copy code
-ADD . /app
+COPY package.json /app/
 
-# Create build
-RUN yarn --production --non-interactive --silent \
-    && yarn build >/dev/null \
+RUN yarn --production --non-interactive --silent
+
+COPY . /app
+
+RUN yarn build >/dev/null \
     && yarn cache clean
 
-USER node
-EXPOSE 3000
-CMD [ "node", "./scripts/serve" ]
+FROM nginx:alpine
```

Comment:
We're trying to use centos/rhel containers across the board, would it be possible to switch to using https://hub.docker.com/r/centos/nginx-112-centos7/ ?

Reply:
Another alternative is using s2i, maybe with https://github.com/jshmrtn/s2i-nodejs-nginx ?

Reply:
Switching should be easy, giving it a go

Reply:
solved! see 7ffb4b3455ea1e7ff548edf246c404c66240912c many thanks @ilyavw